Reflection_Engine/Query/Count.cs

```diff
@@ -39,24 +39,6 @@ public static int ICount<T>(this List<T> list)
         {
             return Item(list as dynamic);
         }
-
-        /***************************************************/
-        /**** Public Methods                            ****/
-        /***************************************************/
-
-        public static int Item<T>(this object obj)
```

Comment:
I think in this case, the name of the method should have been corrected instead of moving the methods to a new file

Reply:
Same as before, this should not be moved but renamed instead